index.html

```diff
@@ -85,16 +76,13 @@ <h1 class="page-header" id="introducing-appium" data-localize="introducing-appiu
 
             <div id="show-demo-video" style="display:none">
                 <iframe width="640" height="360"
-                        src="http://www.youtube.com/embed/1J0aXDbjiUE"
-                        frameborder="0" allowfullscreen></iframe>
-                <iframe width="480" height="360"
-                        src="http://www.youtube.com/embed/oTIEt11cXNk"
-                        frameborder="0" allowfullscreen></iframe>
+                        src="https://www.youtube.com/embed/D5MF3ZLds0Q"
+                        frameborder="0" allowfullscreen></iframe>                
                 <iframe width="640" height="360"
-                        src="http://www.youtube.com/embed/ILKi6B54_sw"
+                        src="https://www.youtube.com/embed/WIuaT6nV9C4"
```

Comment:
@jlipps I am not a big fan of videos at the first place. But there is no harm in keeping them. But I didn't want to have something that was old but rather more recent.